internal/lib/container_server.go

```diff
@@ -309,6 +309,9 @@ func (c *ContainerServer) LoadSandbox(id string) (retErr error) {
 		return err
 	}
 	if err := c.podIDIndex.Add(id); err != nil {
+		if err1 := c.ctrIDIndex.Delete(scontainer.ID()); err1 != nil {
```

Comment:
Can we use a defer instead?

Reply:
Since this is toward the end of the method, I added the Delete() call directly.

If you feel there could be more code added after the if block in the future, I will change this to defer.

Reply:
@mrunalp 
Your comment has been addressed.